private_acl_rules.tf

```diff
@@ -380,3 +429,18 @@ resource "aws_network_acl_rule" "private_egress_to_cool_via_ipa_ports" {
   from_port      = each.value.port
   to_port        = each.value.port
 }
+
+# Allow egress to private IPs via all ports and protocols
```

Comment:
I suggest adding a bit of text here explaining why `172.16.0.0/12` is the only private subnet we care about.  Is that what all the assessors are using for the VMs they spin up on their MacBooks?

Also, these `172.16.0.0/12` private IPs are different than the private IPs in the private subnets in the VPC, but I feel like we're calling them both "private" and that is confusing.  I suggest we find another name for these `172.16.0.0/12` private subnets to reduce confusion.

Reply:
Please take a look at https://github.com/cisagov/cool-assessment-terraform/pull/206/commits/0da00676cce3ca3b212ea163db0f1df9a268ea8e and tell me if that works for you.

Reply:
I'd still like a comment to explicitly state that the `172.16.0.0/12` network is where the assessors are running the VMs on their Macbooks.  "Local VM" could be local to the user's machine or local to the AWS instance and hence is ambiguous, and I won't remember what it means when I revisit this code.

Also, for this to work we require the network on the end user's machine to be set up in a particular way.  (This may be the default way for VMWare on OSX, but on my machine, e.g.,  it would likely be trying to route packets to Docker containers.)  This is a big deal that we shouldn't lose sight of.

For these reasons, I think it makes sense to write a few sentences explaining exactly what is going on here.

Reply:
Yes, that scratches the itch.  Thank you.

Reply:
No, thank you!  Updated in https://github.com/cisagov/cool-assessment-terraform/pull/206/commits/84cb2816feecff60f1eae8d7557755349f19923e.

Reply:
I see commit 84cb281.  Thank you.